<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserSettingsWidget</class>
 <widget class="QWidget" name="UserSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="mainStackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="localUserPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <item>
        <widget class="QWidget" name="userTypeWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="nx::vms::client::desktop::LabelWithHint" name="userTypeLabel">
              <property name="text">
               <string>User Type</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>userTypeComboBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="userTypeComboBox"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>10</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="secondaryStackedWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="newLocalUserPage">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="bottomMargin">
              <number>8</number>
             </property>
             <item>
              <widget class="nx::vms::client::desktop::InputField" name="loginInputField" native="true"/>
             </item>
             <item>
              <widget class="nx::vms::client::desktop::InputField" name="nameInputField" native="true"/>
             </item>
             <item>
              <widget class="nx::vms::client::desktop::InputField" name="emailInputField" native="true"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="nx::vms::client::desktop::PasswordInputField" name="passwordInputField" native="true"/>
           </item>
           <item>
            <widget class="nx::vms::client::desktop::PasswordInputField" name="confirmPasswordInputField" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="newCloudUserPage">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="nx::vms::client::desktop::InputField" name="cloudEmailInputField" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="cloudUserPage">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QnCloudUserPanelWidget" name="cloudPanelWidget" native="true"/>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
     <property name="leftMargin">
      <number>16</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>16</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="roleLabel">
       <property name="text">
        <string>Role</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>roleComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="roleComboBox">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="editRolesButton">
       <property name="text">
        <string>Edit Roles...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QnWordWrappedLabel" name="permissionsLabel" native="true">
       <property name="text" stdset="0">
        <string notr="true">&lt;table&gt;
&lt;tr&gt;&lt;td&gt;&lt;b&gt;3&lt;/b&gt; / 6&lt;/td&gt;&lt;td&gt;&lt;/td&gt;&lt;td&gt;Permissions&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;&lt;b&gt;18&lt;/b&gt; / 91&lt;/td&gt;&lt;td&gt;&lt;/td&gt;&lt;td&gt;Cameras&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;&lt;b&gt;2&lt;/b&gt; / 8&lt;/td&gt;&lt;td&gt;&lt;/td&gt;&lt;td&gt;Layouts&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;&lt;td&gt;&lt;b&gt;0&lt;/b&gt; / 3&lt;/td&gt;&lt;td&gt;&lt;/td&gt;&lt;td&gt;Servers&lt;/td&gt;&lt;/tr&gt;
&lt;/table&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnWordWrappedLabel</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/word_wrapped_label.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::InputField</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/input_field.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::PasswordInputField</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/password_input_field.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::LabelWithHint</class>
   <extends>QLabel</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/widget_with_hint.h</header>
  </customwidget>
  <customwidget>
   <class>QnCloudUserPanelWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/properties/cloud_user_panel_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>roleComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
